rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+max_width = 120
+wrap_comments = true
```

Comment:
This PR contains quite a lot of formatting-only changes. This complicates the review unnecessarily, and usually I would like to review these separately.

Reply:
Agreed, removed